ui/lib/core/addon/templates/components/replication-dashboard.hbs

```diff
@@ -53,7 +53,7 @@
       <AlertBanner
         @title="Syncing in progress"
         @type="info"
-        @secondIconType="loading"
```

Comment:
There is only one usage of the property `secondIconType` , which is "loading". Therefore, I changed this property to a boolean `showLoading` so as not to confuse the possible third icon we can show now when we display a learn-link.